Modules/VMware.vSphere.SsoAdmin/src/test/ConnectDisconnect.Tests.ps1

```diff
@@ -84,6 +84,63 @@ Describe "Connect-SsoAdminServer and Disconnect-SsoAdminServer Tests" {
          $expected.IsConnected | Should Be $false
       }
 
+      It 'Diconnect-SsoAdminServer disconnects the currently connected SSO in case there is 1 SSO server' {
+         # Arrange
+         $expected = Connect-SsoAdminServer `
+               -Server $VcAddress `
+               -User $User `
+               -Password $Password `
+               -SkipCertificateCheck
+
+         # Act
+         Disconnect-SsoAdminServer -server $expected
+
+         # Assert
+         $global:DefaultSsoAdminServers | Should Not Contain $expected
+         $expected.IsConnected | Should Be $false
+      }
+
+      It 'Diconnect-SsoAdminServer does not disconnect if connected to more than 1 SSO server' {
+         # Arrange
+         $expected += @(Connect-SsoAdminServer `
+               -Server $VcAddress `
+               -User $User `
+               -Password $Password `
+               -SkipCertificateCheck)
+         $expected += @(Connect-SsoAdminServer `
+               -Server $VcAddress `
+               -User $User `
+               -Password $Password `
+               -SkipCertificateCheck)
+
+         # Act
+         
+         # Assert
+         {Disconnect-SsoAdminServer} | should -Throw
```

Comment:
You can put the error message as expected as well

Reply:
fixed